Services/Twilio/WorkflowConfiguration.php

```diff
@@ -24,6 +24,24 @@ public static function parse($json) {
 		return json_decode($json);
 	}
 
+	public static function fromJson($json) {
+		$configJSON = json_decode($json);
+		$filters = $configJSON->task_routing->filters;
+		$default_filter = $configJSON->task_routing->default_filter;
+		// aggressive check to see if filter_friendly_name is used
+		if(isset($configJSON->task_routing->filters[0]->filter_friendly_name)) {
```

Comment:
This is true. I initially implemented this as a performance improvement, but better to be through. So if the first filter didn't have a `filter_friendly_name` set, but the second one did, we would miss the conversion. Not super likely, but a possibly.

I've updated this to always convert.